frontend/src/app/modules/group/group-chat/group-chat.component.sass

```diff
@@ -0,0 +1,106 @@
+body
+  background: #eee
+
+.card-body
+    min-height: 350px
+    max-height: 450px
+    overflow-y: scroll !important
+
+.chat-list
+  padding: 0
+  font-size: .8rem
+
+  li
+    margin-bottom: 10px
+    overflow: auto
+    color: #ffffff
+
+  .chat-img
+    float: left
+    width: 48px
+
+    img
+      -webkit-border-radius: 50px
+      -moz-border-radius: 50px
+      border-radius: 50px
+      width: 100%
+      cursor: pointer
+
+  .chat-message
+    -webkit-border-radius: 50px
+    -moz-border-radius: 50px
+    border-radius: 20px
+    background: #3eaaaf
+    display: inline-block
+    padding: 10px 20px
+    position: relative
+    max-width: 350px
+    word-wrap: break-word
+	
+	
+
+    &:before
+      content: ""
+      position: absolute
+      top: 15px
+      width: 0
+      height: 0
+
+    h5
+      margin: 0 0 5px 0
+      font-weight: 600
+      line-height: 100%
+      font-size: .9rem
+      cursor: pointer
+
+    p
+      line-height: 18px
+      margin: 0
+      padding: 0
+
+  .chat-body
+    margin-left: 20px
+    float: left
+    width: 70%
+
+  .in .chat-message:before
+    left: -12px
+    border-bottom: 20px solid transparent
+    border-right: 20px solid #3eaaaf
+
+  .out
+    .chat-img
+      float: right
+
+    .chat-body
+      float: right
+      margin-right: 20px
+      text-align: right
+
+    .chat-message
+      background: #d84848
+      max-width: 350px
+      word-wrap: break-word
+
+      &:before
+        right: -12px
+        border-bottom: 20px solid transparent
+        border-left: 20px solid #d84848
+
+
+button
```

Comment:
Better to style class, but not an element

Reply:
ok, but it was only 1 button on a page so I decided that it's ok
But I will change it if you say so